iphone/Maps/UI/Editor/MWMEditorViewController.mm

```diff
@@ -724,6 +724,18 @@ - (void)fillCell:(UITableViewCell * _Nonnull)cell atIndexPath:(NSIndexPath * _No
   }
 }
 
+- (void)removeAdditionalNameAtIndexPath:(NSIndexPath *)indexPath
+{
+  MWMEditorAdditionalNameTableViewCell * cell = [self.tableView cellForRowAtIndexPath:indexPath];
+  if (!cell)
+    return;
+  [self cell:cell changedText:@""];
+  m_mapObject.RemoveNeedlessNames();
```

Comment:
Is it a cleanup after a name has been set to the empty one? Is there a method to "delete translation", and if no, would it be better to introduce it? Also keep Android in mind where the similar API should be used.

Reply:
Yes! It would be great to clean up the provided language if needed directly.

Reply:
Can you please check the new `RemoveNameForLanguage` method in the `editable_map_object`?